cfme/tests/automate/custom_button/test_generic_objects.py

```diff
@@ -316,3 +316,27 @@ def test_custom_button_expression_evm_obj(appliance, request, setup_obj, button_
             assert not custom_button_group.is_displayed
             setup_obj.add_tag(tag)
             assert button.text in custom_button_group.items
+
+
+@pytest.mark.manual
+def test_custom_button_open_url_evm_obj(setup_obj, button_group):
```

Comment:
Possible to parametrize this test against the different objects? Or is this something you'd like to wait to do until you automate the tests? 

Also could you reference the RFE BZ in here?

Reply:
yup @john-dupuy 
`evm` object need to set as per `object type`. so I need to modify fixture means need a method fixture where i can send object type. I won't like to automate before feature comes in downstream. yup easy to automate for me just waiting.

Reply:
@john-dupuy as you know I like to collect BZ. but this is RFE when just they take decision to add `open url` for other objects. I would not treat it as BZ. Actually `startsin` pointing this. but still if you want let me know.

Reply:
I think better to add let me add this :)

Reply:
I'd say it's optional, I've been adding them just as a reference to help understand the test case, but I wouldn't say that it's necessary.

Reply:
yup totally agree with you... better to have `BZ` collection against test. It helps me lot. just i search `BZ` number in `IDE` and I am getting related tests. personally its help me lot. I don't know about other team member. but i found some of us fallowing `Bugzilla` tag.